overlord/ifacestate/ifacemgr.go

```diff
@@ -59,3 +114,43 @@ func (m *InterfaceManager) Ensure() error {
 func (m *InterfaceManager) Stop() error {
 	return nil
 }
+
+// Interfaces returns object holding a lists of all the plugs and slots and their connections.
+func (m *InterfaceManager) Interfaces() *interfaces.Interfaces {
```

Comment:
As we discussed, these methods shouldn't be here. At least not until we understand why they are here. The manager's task is to actually manage its interface duties, not to export the API of the repository (it will call these methods by itself, in other words).